src/client/js/Panels/ControllerBase/DiagramDesignerWidgetMultiTabMemberListControllerBase.js

```diff
@@ -1808,6 +1868,7 @@ define(['js/logger',
         this._GMEID2ComponentID[gmeID].push(uiComponent.id);
         this._ComponentID2GMEID[uiComponent.id] = gmeID;
 
+        this._delayedConnectionsAsItems = this._delayedConnectionsAsItems || {};
```

Comment:
Couldn't this property be defined in the constructor? Is it declared here too? Or is it set to null at some point?